### ISSUE TITLE
/usr/bin/env: ‘ruby\r’: No such file or directory

### ISSUE THREAD

**Issue by hfnhf** [licensee/licensee#637]
### Describe the bug

Version 9.16.0, docker installation only. Possibly something is missing from the ruby image being used (ruby was updated from 3.2.0 to 3.2.1 on line 1 (the only line changed) in "licensee/licensee/blob/main/Dockerfile" recently)

### Steps to reproduce the behavior

1. ```git clone https://github.com/licensee/licensee && cd licensee```
2. ```docker build . --tag licensee```

3. Two types as example below

Example 1: 
```docker run licensee detect love2d/love --remote```

Example 2: (detecting the license of rails/rails on GitHub):
```docker run licensee detect rails/rails --remote```

result of three steps (running step three twice: both examples have same result both times)
```/usr/bin/env: ‘ruby\r’: No such file or directory```
### Expected behavior

Ruby returns something about the license of the remote project, or some exception from the licensee program rather than something that looks like a docker environment problem.

### Much respect and thanks for your time.

End.

**Comment by manne** [comment]
Same with the command `version`.

**Comment by manne** [comment]
Also for the Git tags `v9.15.3` and `v9.15.2`

**Comment by mlinksva** [comment]
I can't replicate this locally, and [CI](https://github.com/licensee/licensee/blob/c34b1c8f66e808467b862c73d6181e91963f75be/.github/workflows/docker.yml) for docker [passes](https://github.com/licensee/licensee/actions/runs/4159825213/jobs/7196237730). @hfnhf and/or @manne could you say more about your environment or any other info that might help replicate?

**Comment by hfnhf** [comment]
Hi Mike, thanks for reply. For the opening post I was running docker images on Docker Desktop on Windows. I have access to other boxes to get linux installed but will take me a while to setup myself in differen't envs as quite a bit overloaded. Please keep the more-information-needed tag as it is not reproducible on your local.

**Comment by mlinksva** [comment]
Ah, I guess `\r` was a giveaway that it's a on Windows problem.

Seems to be fairly commonplace, many SO questions, other such as https://pauledenburg.com/docker-on-windows-usr-bin-env-bashr-no-such-file-or-directory/

The advice seems to be to convert `\r` to `\n`, though no files in this repo need that -- only `spec/fixtures/crlf-bsd/LICENSE.md` exits and wouldn't be encountered except when running tests.

Any chance you're editing a file with an editor that defaults to `\r`? Just throwing out a guess!

**Comment by manne** [comment]
👍Changing the line endings to `\r` (or `git config core.autocrlf false`) fixed the issue

**Comment by mlinksva** [comment]
Great. I guess this is a documentation issue then, as I don't see a way to address automatically?

**Comment by hfnhf** [comment]
Yes this could be addressed with documentation about WSL2. But I'm not too familiar to feedback as to when the linending is getting mangled by Windows. 
I can see in the docker build file that docker build "apt" commands occur in linux so maybe is it getting mangled by the original git clone command. In any case if you want me to close the issue to get it out of the way instead of closing after documentation update, please let me know (thanks for having a look).

**Comment by mlinksva** [comment]
Fine with me to leave open until there's a documentation update. Or if there's a better fix, I welcome that.
